LocalSettings.php

```diff
@@ -6475,6 +6475,49 @@
 		'default' => true,
 	],
 
+	// WikidataPageBanner
+	'wgWPBImage' => [
+		"default" => null,
```

Comment:
```suggestion
		'default' => null,
```

and for all the other strings as well :)